### ISSUE TITLE
What's the name of our project?

### ISSUE THREAD

**Issue by rbrunner7** [seraphis-migration/wallet3#15]
Seems to me we should soon agree on a name of the project at hand, building a `wallet2` successor to support Seraphis and Jamtis. If only for such mundane things like naming repositories, naming the new folder in the Monero source code tree for that wallet, and similar.

This repository here where the issues are located so far is called *wallet3*. When I created it a few months ago I used that merely as a "working title". But anyway, with the existing wallet called *wallet2*, *wallet3* of course is natural to derive. So it's one candidate for a project name.

We have now a *Seraphis library*, written by @UkoeHB. This also leads quite naturally to a project name of *Seraphis wallet*. More descriptive than *wallet3*, but also quite a bit longer.

I am not sure that unrelated names like e.g. *Wallet NG* with "NG" standing for "next generation" are a good way to go, but other people may have a different opinion.

Please voice your opinion, and let your own proposals flow!

**Comment by UkoeHB** [comment]
Longer is not a bad thing, I think `seraphis_wallet` is a good term, at least temporarily (or `sp_wallet`).

**Comment by DangerousFreedom1984** [comment]
I agree with koe. Almost all the domain names in the seraphis lib start with sp_. We could name the wallet just Seraphis Wallet and if we need domain separators we could use spw_.

**Comment by One-horse-wagon** [comment]
It is a bit confusing having two big names--Jamtis and Seraphis.  Paring it down to just Seraphis wallet would be my vote.
